sympy/core/tests/test_equation.py

```diff
@@ -0,0 +1,122 @@
+from sympy import var, integrate, simplify, expand, factor, log, \
+    diff, FiniteSet, Matrix, exp, collect
+from sympy import Equality
+from sympy import Equation, Eqn
+from sympy import latex
+from sympy.testing.pytest import raises, warns
+
+
+def test_define_equation():
+    a, b, c = var('a b c')
+    raises(TypeError, lambda: Eqn(FiniteSet(a), FiniteSet(b, c)))
+    with warns(UserWarning):
+        assert Eqn(1, 0) == Equation(1, 0, check=False)
```

Comment:
`Eqn` and `Equation` are the same. Mixing them like this creates a false impression that the distinction is significant.

Reply:
Switched to using `Equation` except to verify that the `Eqn` synonym is valid.
Commit 4dc62ee7